### ISSUE TITLE
Making the Dashboard.

### ISSUE THREAD

**Issue by goliakshay357** [Canvasbird/canvasboard#68]
A dashboard to view all the folders of a user ( JSON Response from backend ). 

![image](https://user-images.githubusercontent.com/32712438/94339967-bd997580-001b-11eb-8769-1121a8c1c76a.png)

As soon as a user logs in, **routelink** will be directed to dashboard.

**Comment by geeks5551** [comment]
hey can I contribute to the above issue?i.e integrating JSON APIs with the current functionalities

**Comment by goliakshay357** [comment]
@geeks5551 Sure :+1: 
We need to remove the existing code in that component and change it accordingly.
We are open for suggestions and Improvements. 

Can you please join our IRC Channel for further discussion?
Link: https://gitter.im/canvasbird/canvasboard-frontend 
Thanks.

**Comment by goliakshay357** [comment]
@geeks5551 . Are you working on this issue?

**Comment by jawsvk** [comment]
@goliakshay357 I am also interested to tackle this. Could you assign it to me? Or is @geeks5551 working on this? It seems like he is interested in a lot of issues in this repo

**Comment by goliakshay357** [comment]
@jawsvk Sure :+1: 
You can take up this issue

Thanks

**Comment by goliakshay357** [comment]
@jawsvk, can you please join our IRC Channel for further discussion?
Link: https://gitter.im/canvasbird/canvasboard-frontend
Thanks.

**Comment by goliakshay357** [comment]
@jawsvk are you working on this issue?

**Comment by jawsvk** [comment]
> @jawsvk are you working on this issue?

@goliakshay357 yes, it is still on my radar, but as I an quite occupied with work the last 2 weeks, I only have time to focus on this on the weekends. Will raise a PR by this weekend.

Is that alright?

**Comment by goliakshay357** [comment]
Yeah sure... 
Anytime :)
